native/cocos/editor-support/spine-wasm/Vector2.cpp

```diff
@@ -0,0 +1,48 @@
+#include "Vector2.h"
+#include <math.h>
+namespace spine {
+
+Vector2::Vector2(): x(0), y(0) {
```

Comment:
Copy from `../spine-creator-support/Vector2.cpp` to make `spine-wasm` project independent.

Reply:
Will it conflict with spine?

Reply:
Vector2.h/.cpp was not the code in `spine-runtime`.
`spine-creator-support` directory is for JSB and `spine-wasm` is for WASM.
We should not include a file (Vector2.h/.cpp) in `spine-creator-support` directory since it is used for JSB.